app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] In keeping with the ERD Diagram, the user should have `posts`, `comments`, and `likes` through the `author_id` foreign key. Kindly implement that.

  Here is an hint:
  ```
  has_many :posts, foreign_key: 'author_id'
  ...